docs/source/api/letsencrypt_dnsrecord.rst

```diff
@@ -0,0 +1,66 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-letsencrypt-dnsrecord:
+
+*************************
+``letsencrypt/dnsrecord``
+*************************
+
+.. versionadded:: 1.4
+
+``GET``
+========
+Gets DNS challenge records for Let's Encrypt DNS challenge for a specified fqdn.
```

Comment:
"fqdn" is an initialism (and an `abbr`, if you're so inclined), should be capitalized.

Reply:
done